src/pkg/packaging/dir.proj

```diff
@@ -158,7 +158,9 @@
   </PropertyGroup>
   <Target Name="GenerateInstallers" DependsOnTargets="$(InstallerDependsOn)" />
 
-  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage">
+  <Import Project="$(ProjectDir)src/managed/CommonManaged.props" />
+  <Target Name="GenerateNugetPackages" 
```

Comment:
This whole file doesn't need to be changed.  Setting `IncludeSymbols` and `Serviceable` in the CommonManaged.props file is enough to get `dotnet pack` to do the right thing.